### PR TITLE
fix(content): th align-left default, allow override

### PR DESCRIPTION
**Pull request by zchsh**
🎟️ [Asana Task]()
🔍 [Preview Link](https://react-components-git-{branch-slug}-hashicorp.vercel.app)

---

## Description

Describe this pull request: what is it aiming to achieve? What should someone reviewing this PR know? If this PR includes visual changes, consider including before / after screenshots, or an animated gif showing interactions.

### PR Checklist 🚀

Items in this checklist may not may not apply to your PR, but please consider each item carefully.

- [ ] Add Asana and Preview links above.
- [ ] Conduct thorough self-review.
- [ ] Add or update tests as appropriate.
- [ ] Conduct reasonable cross browser testing for both compatibility and responsive behavior (We have a [Sauce Labs](https://app.saucelabs.com/) account for this, if you don't have access, just ask!).
- [ ] Conduct reasonable accessibility review (use the [WAS](https://accessible.org/Web-Accessibility-Standards-WAS-2.pdf) as a guide or an [axe browser plugin](https://www.deque.com/axe/) until we establish more formal checks).
- [ ] Identify (in the description above) and document (add Asana tasks on [this board](https://app.asana.com/0/1100423001970639/list)) any technical debt that you're aware of, but are not addressing as part of this PR.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1201999608008890

### CODE REVIEW

File: packages/content/styles/table.module.css [hunk 1]
```diff
@@ -11,6 +11,10 @@
     max-width: 100%;
     margin-bottom: 20px;
 
+    & th:not([align]) {
```
Comment (BRKalow) [inline packages/content/styles/table.module.css:14]: 🪄 🧙

**Comment by zchsh** [comment]
I've proven this out in two places:
- https://github.com/hashicorp/nomad/pull/12581 - see 
- https://github.com/hashicorp/dev-portal/pull/322 - no way to make local content changes, but here are some screenshots:
    - Does not show up if `align` property is set
    - CleanShot 2022-04-15 at 02 04 07@2x
    - Shows up by default
    - CleanShot 2022-04-15 at 02 04 17@2x

**Comment by zchsh** [comment]
@BRKalow I'd been lagging on testing this, but just took a shot at it, and thankfully it looks good to go 🚀

Let me know if you notice anything off - one thing that was bothering me was I didn't automatically get a `docs-page` canary release, even though it depends on `content` 🤔 So I'm little worried we'll have to do a dependency dance of some kind to get this rolled out... but hopefully not 🤞

**Review by BRKalow — APPROVED** [review]
👍

**Pull request merged**